common/macresman.cpp

```diff
@@ -357,10 +347,10 @@ bool MacResManager::loadFromMacBinary(SeekableReadStream &stream) {
 
 		uint32 dataSizePad = (((dataSize + 127) >> 7) << 7);
 		// Files produced by ISOBuster are not padded, thus, compare with the actual size
-		//uint32 rsrcSizePad = (((rsrcSize + 127) >> 7) << 7);
+		uint32 rsrcSizePad = (((rsrcSize + 127) >> 7) << 7);
 
 		// Length check
-		if (MBI_INFOHDR + dataSizePad + rsrcSize <= (uint32)stream.size()) {
+		if (MBI_INFOHDR + dataSizePad + rsrcSizePad <= (uint32)stream.size()) {
```

Comment:
And that change is also incorrect because it means that macbinary created by ISOBuster will fail to load.
Combined with the previous issue I pointed above it means that the commit as it currently is will likely break all macbinary files. The check here should be the same check as in `isMacBinary()`.